bundles/org.eclipse.jface/src/org/eclipse/jface/resource/JFaceResources.java

```diff
@@ -583,6 +584,23 @@ public static void setFontRegistry(FontRegistry registry) {
 		fontRegistry = registry;
 	}
 
+
+	/**
+	 * Creates a local registry that wraps the ResourceManager for the current
+	 * display. Anything allocated by this registry will be automatically cleaned up
+	 * with the given control is disposed. Note that registries created in this way
+	 * should not be used to allocate any resource that must outlive the given
+	 * control.
+	 *
+	 * shortcut for
+	 * <code>LocalResourceManager(JFaceResources.getResources(), owner)</code>
+	 *
+	 * @param owner control whose disposal will trigger cleanup of everything in the
+	 *              registry.
+	 */
+	static LocalResourceManager managerFor(Control owner) {
```

Comment:
Why is this package private?

Reply:
> Why is this package private?

Thanks for the catch. Will be fixed with 
https://github.com/eclipse-platform/eclipse.platform.ui/pull/1218